### PR TITLE
Add instruction to set up database locally

### PR DESCRIPTION
**Pull request by jamestran201**
I ran into some trouble setting up the DB locally so I thought this might be helpful for new devs as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Install [Postgres](https://www.postgresql.org/download/) (on OS X you can use `b
 CREATE USER feedadoc WITH SUPERUSER PASSWORD 'password1';
 ```
 
+2. Set the database url manually or add this command to a file like `.bashrc`:
+
+```
+export DATABASE_URL="postgresql://localhost"
```
Comment (cantino) [inline README.md]: This is documented in the Docker section. Maybe add a comment [there](https://github.com/feedadoc/feedadoc/pull/88/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38)?

Comment (cantino) [reply README.md]: Or, you could add a `## Troubleshooting` section.

The third option is that we suggest that users make a `.env` file and we add [dotenv](https://github.com/bkeepers/dotenv) to the Gemfile. Do you have a preference?

Comment (jamestran201) [reply README.md]: Oh my bad, I didn't try running the command in the Docker section. I don't really have a preference. 

If the steps are the same, then how about we tell them to refer to step 2 in the Docker section after completing step 1?

If you think that dotenv is suitable, then I can add that, too.

Comment (cantino) [reply README.md]: Were you using Docker, or just running this locally with a local postgres process?

Comment (jamestran201) [reply README.md]: I wasn't using Docker, just ran with a local DB.

**Comment by tuntisz** [comment]
Without knowing the root cause of why/when this is needed, I'm hesitant to say this should go in the readme. Do you know under what circumstances one would need to do this? For Docker, it's a matter of socket access.

Instead of updating the docs, perhaps we can try to catch ActiveRecord connection failure exceptions and raise with a message that suggests overriding this var?

**Comment by jamestran201** [comment]
I'll leave the doc as is since adding the step doesn't help clarify things and I'm not really sure how to intercept the initial call to the DB. It looks like setting `ENV['DATABASE_URL']` is a common step in setting up a Rails app. I should have known that 😅.

**Pull request closed without merging**